src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,10 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+
+  margin: 0 auto 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.